src/module/array.js

```diff
@@ -1,28 +1,39 @@
-const array = [
-  {
-    description: 'sweep the floor',
-    completed: true,
-    index: 1,
-  },
+import Tasks from './tasks.js';
+import display from './display.js';
 
-  {
-    description: 'wash my clothes',
-    completed: true,
-    index: 4,
-  },
+export default class Arr {
+  constructor() {
+    this.todos = [];
+  }
 
-  {
-    description: 'brush my teeth',
-    completed: true,
-    index: 2,
-  },
+  addTodo(desc, main) {
+    const fresh = new Tasks(desc, this.todos.length + 1);
+    this.todos.push(fresh);
+    this.todos.map((todo) => display(todo, main));
+  }
 
-  {
-    description: 'clean the house',
-    completed: true,
-    index: 3,
-  },
+  removeTodo(id) {
+    this.todos = this.todos.filter((td) => td.index !== id);
+  }
 
-];
+  changeindex() {
+    this.todos.forEach((todo, index) => {
+      todo.index = index + 1;
+    });
+  }
```

Comment:
#### You did very well on this project so far. I like your UI set very much. Well done 👏 . But there is a little bug when I try to delete a task.
I recorded the video below to better explain what I will try to explain in writing. Please see the video link below 👇 
https://www.loom.com/share/4691396e900f4cc0ade242f747304910

- [ ] In your project, I could add, edit and read tasks. Bt the delete task did not work as expected. When the delete button of a task is clicked, it deletes the item directly above it. Basically, the wrong item is deleted. Please check for this bug and fix it.